org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp

```diff
@@ -224,6 +224,8 @@
 	display: flex;
  	justify-content: center;
  	align-items: center;
+ 	margin: 20px 0px 20px 0px;
+ 	height: -webkit-fill-available;
```

Comment:
this wouldn't work with firefox. what are you trying to achieve here ? have you try flex-grow instead ?

Reply:
Fixed